docs/protocol/extension_trace.md

```diff
@@ -78,7 +78,7 @@ see trace setting "log_frequency" below for detail.
 - "trace_level" : the trace level. "OFF" to disable tracing,
 "TIMESTAMPS" to trace timestamps, "TENSORS" to trace tensors.
 This value is an array of string where user may specify multiple levels to
-trace multiple informations.
+trace multiple information.
```

Comment:
I think it is ok to accept this change for now. I can make a note to rephrase this in a follow-up PR